streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -308,6 +309,20 @@ public void putAll(final List<KeyValue<Bytes, byte[]>> entries) {
         }
     }
 
+    @Override
+    public <PS extends Serializer<P>, P> KeyValueIterator<Bytes, byte[]> prefixScan(final P prefix, final PS prefixKeySerializer) {
```

Comment:
Please put the second parameter on a new line and align the two parameters like:
```suggestion
    public <PS extends Serializer<P>, P> KeyValueIterator<Bytes, byte[]> prefixScan(final P prefix, 
                                                                                    final PS prefixKeySerializer) {
```
(On GitHub the suggestions seems a bit strange)